CONTRIBUTING.md

```diff
@@ -137,9 +144,61 @@ The [packages/](packages/) directory contains the following public/published pac
 ### Internal packages
 
 The [packages/](packages/) directory contains the following internal packages:
+
 * [caliper-publish](packages/caliper-publish/): Utility CLI for publishing Caliper to NPM and DockerHub.
 * [caliper-tests-integration](packages/caliper-tests-integration/): Collection of CI integration tests.
 
+### Testing Methodologies
+
+This section outlines the testing methodologies that this project follows, including both unit-level and integration-level testing.
+
+#### Unit Level Testing
+
+Unit testing focuses on testing individual packages in isolation. The tests are typically located within a test folder at the same level as the corresponding lib folder, mirroring the structure of the lib folder for easy navigation. This setup ensures that each component of the code is verified independently, confirming that it behaves as expected without interference from other parts of the system.
+
+#### Integration Level Testing
```

Comment:
This is fine as a starting point, we will need to expand on this level of testing in the future